### ISSUE TITLE
Is this ever going to be opensource?

### ISSUE THREAD

**Issue by poudelprakash** [LogRocket/logrocket#5]

**Comment by frank-dspeed** [comment]
@poudelprakash i don't think so but i already preparing a opensource version already reverseed it:

I will update you here it is using TraceKit JS and some bindings to send data with it
